azkaban-web-server/src/main/java/azkaban/webapp/servlet/LoginAbstractAzkabanServlet.java

```diff
@@ -276,7 +277,13 @@ protected void doPost(final HttpServletRequest req, final HttpServletResponse re
       throws ServletException, IOException {
     Session session = getSessionFromRequest(req);
     this.webMetrics.markWebPostCall();
-    logRequest(req, session);
+    if (allowedPostRequest(req)) {
+      logRequest(req, session, true);
```

Comment:
i would prefer having allowedPostRequest inside logRequest so that we don't need add a boolean argument to logRequest.